src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -354,6 +374,25 @@ private function refreshUsesCache(array $uses): void
         }
     }
 
+    private function isReservedIdentifier(string $symbol): bool
+    {
+        if (str_contains($symbol, '\\')) { // optimization only
+            return false;
+        }
+
+        if ((new TypeAnalysis($symbol))->isReservedType()) {
+            return true;
+        }
+
+        foreach ($this->reservedIdentifiers as $reservedIdentifiers) {
+            if (isset($reservedIdentifiers[$symbol])) {
```

Comment:
After quick look it's not super clear why `$this->reservedIdentifiers` is iterated as `$reservedIdentifiers` (plural again), maybe something like this could help:

```suggestion
        foreach ($this->reservedIdentifiers as $scopeReservedIdentifiers) {
            if (isset($scopeReservedIdentifiers[$symbol])) {
```

I mean, it should be more readable that these identifiers are scope-narrowed. Naming is just a proposal 🙂.

Reply:
I agree. The iterated plural variable should be kept, any idea how to name the property better?

Reply:
addressed